pubsub/gcppubsub/gcppubsub_test.go

```diff
@@ -65,27 +66,41 @@ func newHarness(ctx context.Context, t *testing.T) (drivertest.Harness, error) {
 	return &harness{done, pubClient, subClient}, nil
 }
 
-func (h *harness) MakeTopic(ctx context.Context) (driver.Topic, error) {
-	dt := openTopic(ctx, h.pubClient, projectID, topicName)
-	return dt, nil
+func (h *harness) MakeTopic(ctx context.Context) (dt driver.Topic, cleanup func(), err error) {
+	topicName := fmt.Sprintf("test-topic-%d", time.Now().UnixNano())
```

Comment:
It works because you're picking the same random number every time ("Top-level functions, such as Float64 and Int, use a default shared Source that produces a deterministic sequence of values each time a program is run."). I thought the point was to pick a different value on each run.

Reply:
The point is to make sure different subscriptions in the tests have different names. They don't have to be different across runs since they get deleted when the test ends, although that does introduce some risk of flaking in record mode since terminating a test could leave resources on the server that would interfere with subsequent runs.

I agree that rand isn't a great choice here since it doesn't really convey the intent. I'll try something else.